### ISSUE TITLE
About blank

### ISSUE THREAD

**Issue by Chen2022** [adizanni/floor3d-card#105]
Set the temperature to be normal, and then add a blank after water leakage detection
binary_sensor.water_leak_sensor_158d00044be77b,Not supported? Blank after setting
<img width="1068" alt="001" src="https://user-images.githubusercontent.com/70124937/165913898-8cbf2ca9-a741-426c-b8b0-4ca7a156a184.png">
<img width="1055" alt="02" src="https://user-images.githubusercontent.com/70124937/165913911-65b435a2-7e1b-4b40-9aec-203fa207b0f5.png">
<img width="1014" alt="003" src="https://user-images.githubusercontent.com/70124937/165913917-783d1adc-0c2d-46b6-a0b3-e11348395877.png">

**Comment by Chen2022** [comment]
It returns off and on,
[[[ if ($entity = off) { "hot" } else { "cool" } ]]]
After I set it like this, it is still blank

**Comment by adizanni** [comment]
if the state of the entity is on or off (please verify in the developers tools -> states), then the right syntax is the following:

```js
[[[ if ($entity == "on") { "hot" } else { "cool" } ]]]
```

**Comment by Chen2022** [comment]
Yes, it's on or off. Adding direct display is normal. After I set it, 3D card will not be displayed.
.
- entity: binary_sensor.water_leak_sensor_158d00044be77b
    type3d: color
    object_id: room_58_280
    colorcondition:
      - state: hot
        color: red
    entity_template: '[[[ if ($entity == "on") { "hot" } else { "cool" } ]]]'
 
<img width="994" alt="001" src="https://user-images.githubusercontent.com/70124937/165948692-d55e77d1-aba1-497b-83ee-1107721b60c7.png">
<img width="425" alt="22" src="https://user-images.githubusercontent.com/70124937/165949490-1e4bce38-9f18-40ca-b670-6c086b3ba7bf.png">

**Comment by Chen2022** [comment]
Yes, thank you very much
